### PR TITLE
Adiep new app header

### PR DESCRIPTION
**Pull request by anndiep-sfcc**
## All Pull Requests:
* [ ] Have you considered Security (e.g. XSS)?
* [ ] Have you considered desktop, mobile and tablet form-factors?
* [ ] Have you considered Accessibility best practices?

### Code
* [ ] Are the commits squashed into one commit?
* [ ] Is the branch name and commit message following team convention?
* [ ] Is the code linted?
* [ ] Are all open issues, questions and concerns by Reviewers answered, resolved and reviewed?

### Quality Assurance
* [ ] If applicable, are Unit Tests written and all tests are passing?
* [ ] If applicable, are Integration Tests written and all tests are passing?
* [ ] If applicable, are UI Automation Tests written and all tests are passing?

### Documentation
* [ ] Are there meaningful code comments included?
* [ ] If applicable, are test cases written?
* [ ] If applicable, is Change Log updated?
* [ ] If applicable, are UI Texts reviewed by Documentation?

### Security
* [ ] If applicable, has Security reviewed this code?
* [ ] If applicable, is a 3PP request submitted?

### CODE REVIEW

File: packages/storefront-lwc/src/modules/commerce/contextHeading/contextHeading.scss [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  
+  .sub-header-content {
+    background-color: white;
+    width: auto;
+    max-width: 90%;
+    overflow: hidden;
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
```
Comment (blittle) [inline packages/storefront-lwc/src/modules/commerce/contextHeading/contextHeading.scss:36]: Is the position absolute, transform, left, and top to center the content? There are easier ways to do this. For example, `display: flex; justify-content: center; align-items: center;`

File: packages/storefront-lwc/src/modules/commerce/basket/basket.html [hunk 2]
```diff
@@ -5,6 +5,7 @@
     For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
 -->
 <template>
+    <commerce-context-heading searchtext={searchtext} headertext="Your Basket"></commerce-context-heading>
```
Comment (blittle) [inline packages/storefront-lwc/src/modules/commerce/basket/basket.html:8]: I could be wrong, but I don't think that `searchtext={searchtext}` is necessary?

File: packages/storefront-lwc/src/modules/commerce/productSearchResults/productSearchResults.js [hunk 3]
```diff
@@ -218,4 +224,20 @@ export default class ProductSearchResults extends LightningElement {
...
     };
+
+    /**
+     * Set the apropriated text in the heading.
+     * @param showHeader A flag indicating whether or not to show the heading
+     */
+    createHeader(showHeader) {
+        if (showHeader && this.hasProducts()) {
+            this.searchtext = 'Search result for';
+            this.headertext = this.query;
```
Comment (blittle) [inline packages/storefront-lwc/src/modules/commerce/productSearchResults/productSearchResults.js:235]: Maybe nitpicky, but what if someone used this component for something that wasn't for search, and still wanted to use `searchtext`? Is there a better word for the property? `headertext` and `secondarytext`? I don't know. The `headertext` is what is on mobile and desktop, whereas the other is what disappears in mobile.

File: packages/storefront-lwc/src/modules/commerce/contextHeading/contextHeading.scss (hunk 1 above)
Comment (blittle) [reply packages/storefront-lwc/src/modules/commerce/contextHeading/contextHeading.scss:36]: Flexbox (and css grid) really change the way CSS is written. I highly recommend taking some time to learn it. Here is a free (and fun) course that a friend of mine built: https://geddski.teachable.com/p/flexbox-zombies

Comment (anndiep-sfcc) [reply packages/storefront-lwc/src/modules/commerce/contextHeading/contextHeading.scss:36]: Thank you @blittle for sharing link! Taking the tutorial.  It's fun, as you said, and educational!

File: packages/storefront-lwc/src/modules/commerce/basket/basket.html (hunk 2 above)
Comment (anndiep-sfcc) [reply packages/storefront-lwc/src/modules/commerce/basket/basket.html]: removed searchtext.

File: packages/storefront-lwc/src/modules/commerce/productSearchResults/productSearchResults.js (hunk 3 above)
Comment (anndiep-sfcc) [reply packages/storefront-lwc/src/modules/commerce/productSearchResults/productSearchResults.js:235]: renamed to property to headertextprefix.

File: packages/storefront-lwc/src/modules/commerce/contextHeading/contextHeading.scss (hunk 1 above)
Comment (anndiep-sfcc) [reply packages/storefront-lwc/src/modules/commerce/contextHeading/contextHeading.scss:36]: Spoke to @blittle.  He is ok with this styling.

File: packages/storefront-lwc/src/modules/commerce/productSearchResults/productSearchResults.js [hunk 4]
```diff
@@ -26,6 +26,9 @@ export default class ProductSearchResults extends LightningElement {
     };
     sortRuleValue = '';
 
+    headertextprefix = '';
+    headerText = '';
```
Comment (blittle) [inline packages/storefront-lwc/src/modules/commerce/productSearchResults/productSearchResults.js:30]: Every where else this variable isn't camel cased

Comment (anndiep-sfcc) [reply packages/storefront-lwc/src/modules/commerce/productSearchResults/productSearchResults.js:30]: Made the correction. Thx!

**Pull request merged**